src/main/resources/com/google/api/codegen/py/main.snip

```diff
@@ -24,7 +24,7 @@
     {@serviceClass(service)}
 @end
 
-@private licenseSection(service)
+@private renderLicenseSection(service)
```

Comment:
The name was changed to prevent conflict with `licenseSection` in py/common.snip. When we fully migrate to MVVM, `renderLicenseSection` will be removed.